src/utils/guildRole.ts

```diff
@@ -84,3 +86,34 @@ export async function removeGuildRole(details: memberGroupRole, env: env) {
     return INTERNAL_SERVER_ERROR;
   }
 }
+
+export async function getGuildRoles(env: env): Promise<Array<GuildRole>> {
+  const guildDetailsUrl = `${DISCORD_BASE_URL}/guilds/${env.DISCORD_GUILD_ID}`;
+
+  const response = await fetch(guildDetailsUrl, {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bot ${env.DISCORD_TOKEN}`,
+    },
+  });
```

Comment:
please use it inside a try catch

Reply:
These functions are used in controllers where there is proper error handling. Hence any error thrown here will be caught in the controller and a proper response will be sent.

Hence I have decided not to wrap in try-catch unnecessarily. Let me know if you want to connect for this and discuss on the benefits and demerits of this decision. 😅

Reply:
let's say tomorrow a handler needs to call 50 similar functions without proper handling in each case how would you know which fetch call/function has failed?

Reply:
I understand what are you trying to say. Though I don't completely agree, I will make changes as per your suggestion so the PR can move ahead.

Reply:
Hey @RitikJaiswal75, 
I added a try-catch now in the utils function. 

Thank you for your feedback 🤝